ops/index.js

```diff
@@ -85,13 +100,18 @@ async function migrate(web3) {
   const Token = new web3.eth.Contract(DAOToken.abi, undefined, opts);
   const token = await Token.deploy({
     data: DAOToken.bytecode,
-    arguments: ["TEST","TST",1000000000]
-    }).send();
+    arguments: ["TEST", "TST", 1000000000]
+  }).send();
+
+  const gpToken = await Token.deploy({
```

Comment:
why gpToken ? can we just use token for gp ?

Reply:
No, it have the same problem as with using the same reputation. It makes the DAOToken test fail

Reply:
so just deploy that locally at the GP tests

Reply:
I can't as the GP requires that in its constructor